xxh3.h

```diff
@@ -84,12 +91,15 @@ ALIGN(64) static const U32 kKey[KEYSET_DEFAULT_SIZE] = {
 };
 
 
-
-
-XXH_FORCE_INLINE U64
+#if defined(__GNUC__) && defined(__i386__)
+/* GCC is stupid and tries to vectorize this.
+ * This tells GCC that it is wrong. */
+__attribute__((__target__("no-sse")))
+#endif
+static U64
 XXH3_mul128(U64 ll1, U64 ll2)
 {
-#if defined(__SIZEOF_INT128__) || (defined(_INTEGRAL_MAX_BITS) && _INTEGRAL_MAX_BITS >= 128)
+#if 0 && defined(__SIZEOF_INT128__) || (defined(_INTEGRAL_MAX_BITS) && _INTEGRAL_MAX_BITS >= 128)
```

Comment:
I assume the `0 &&` here was there for tests ?

Reply:
I have no idea what you are talking about.